eng/pipelines/libraries/execute-linker-tests-steps.yml

```diff
@@ -0,0 +1,4 @@
+  steps:
+    # Execute tests
```

Comment:
Might be worth adding a step to publish test results into the Azure DevOps tab, we could open an issue for that though.

Reply:
Yeah, I think that would be definitely great to have, but we can keep that as part of a subsequent PR